docs/LOGS.md

```diff
@@ -0,0 +1,130 @@
+- [Introduction](#introduction)
+  - [Log Types](#log-types)
+  - [Fields](#fields)
+    - [Access Fields](#access-fields)
+    - [Backend Fields](#backend-fields)
+    - [Daemon Fields](#daemon-fields)
+  - [Features](#features)
+
+# Introduction
+
+_Logs_ are the information being printed in the console upon the execution of Couper. There are a few different [Log Types](#log-types) and several different [Fields](#fields) containing useful information given in said _Logs_ to be remembered, but you can adjust the _Logs'_ appearance and verbosity with a variety of [Features](#features) to make sure that the information you seek is available as easily as possible.
+
+These _Logs_ are however still subject to change. We aspire to make Couper as stable as possible and that includes the _Logs_, so until we believe to have achieved that goal you should expect to see some changes.
+
+## Log Types
+
+| Type                | Description                                                                                                                                     |
+| :------------------ | :---------------------------------------------------------------------------------------------------------------------------------------------- |
+| `couper_access`     | Provides information about the frontend side of things. For information about its [Fields](#fields), see [Access Fields](#access-fields).       |
+| `couper_access_tls` | Provides information about connection to TLS proxies. For information about its [Fields](#fields), see [Access Fields](#access-fields). |
+| `couper_backend`    | Provides information about the backend side of things. For information about its [Fields](#fields), see [Backend Fields](#backend-fields).      |
```

Comment:
Add some additional information like upstream request / response states. Origin informations are useful for example.